ui/admin.html

```diff
@@ -106,7 +106,6 @@
    <!--endbuild-->
 
   <script src="vendor/ngReact/ngReact.min.js"></script>
-  <script src="https://login.persona.org/include.js"></script>
```

Comment:
Yey for getting rid of the last third party JS include! Will make our CSP work simpler in the future :-)